source/common/tracing/zipkin/zipkin_json_field_names.h

```diff
@@ -0,0 +1,31 @@
+#pragma once
+
+#include <string>
+
+namespace Zipkin {
+
+class ZipkinJsonFieldNames {
+public:
+  static const std::string SPAN_TRACE_ID;
+  static const std::string SPAN_PARENT_ID;
+  static const std::string SPAN_NAME;
+  static const std::string SPAN_ID;
+  static const std::string SPAN_TIMESTAMP;
+  static const std::string SPAN_DURATION;
+  static const std::string SPAN_ANNOTATIONS;
+  static const std::string SPAN_BINARY_ANNOTATIONS;
+
+  static const std::string ANNOTATION_ENDPOINT;
+  static const std::string ANNOTATION_TIMESTAMP;
+  static const std::string ANNOTATION_VALUE;
+
+  static const std::string BINARY_ANNOTATION_ENDPOINT;
+  static const std::string BINARY_ANNOTATION_KEY;
+  static const std::string BINARY_ANNOTATION_VALUE;
+
+  static const std::string ENDPOINT_SERVICE_NAME;
+  static const std::string ENDPOINT_PORT;
+  static const std::string ENDPOINT_IPV4;
+  static const std::string ENDPOINT_IPV6;
```

Comment:
These are also all subject to the static initialization order fiasco. Read about this at:
* https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables
* https://isocpp.org/wiki/faq/ctors#static-init-order
* https://isocpp.org/wiki/faq/ctors#static-init-order-on-first-use
It might make sense to make these `const char[]` POD C strings. Or add some macros to reduce the boiler plate around initialization-on-first-use. 

I'm not bringing this up to be picky, but because this will fail ASAN when we switch on the check for this and it's really like playing with fire, since any small changes to the compiler or phase of the moon might lead to crashes during construction or destruction.